hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/config/HoodieRealtimeConfig.java

```diff
@@ -17,26 +17,20 @@
 
 package org.apache.hudi.hadoop.config;
 
-/**
- * Class to hold props related to Hoodie RealtimeInputFormat and RealtimeRecordReader.
- */
-public final class HoodieRealtimeConfig {
+import org.apache.hudi.common.config.ConfigProperty;
+import org.apache.hudi.common.config.HoodieConfig;
+
+public class HoodieRealtimeConfig extends HoodieConfig {
```

Comment:
`HoodieRealtimeConfig` no longer exists and these configs are moved to `HoodieMemoryConfig`, `HoodieCommonConfig` and `HoodieReaderConfig`.